index.js

```diff
@@ -49,6 +49,10 @@ function copy(src) {
 			dst = { message: src.message };
 		} else if (isBoolean(src) || isNumber(src) || isString(src)) {
 			dst = Object(src);
+		} else if (src instanceof Buffer) {
+			dst = Buffer.alloc(src.length, src);
+		} else if (src instanceof Uint8Array || src instanceof ArrayBuffer) {
+			dst = src.slice(0);
```

Comment:
similarly, instanceof can't be relied on - but also what about the other typed arrays?

Reply:
Not sure. Frankly. I'm working in a Node.js environment, so I'm primarily concerned about making `Buffer` work. Should I remove this case completely instead?

Reply:
I have now removed the changes related to the browser types. It could be addressed in another PR.